tests/10-create.js

```diff
@@ -6,13 +6,16 @@ import chai from 'chai';
 import {
   checkDataIntegrityProofFormat
 } from 'data-integrity-test-suite-assertion';
-import {filterByTag} from 'vc-api-test-suite-implementations';
+import {endpoints} from 'vc-api-test-suite-implementations';
 import {generateTestData} from './vc-generator/index.js';
 import {klona} from 'klona';
 import {v4 as uuidv4} from 'uuid';
 // only use implementations with `Ed25519 2020` issuers.
 const tag = 'Ed25519Signature2020';
-const {match, nonMatch} = filterByTag({tags: [tag], property: 'issuers'});
+const {
+  match,
+  nonMatch
+} = endpoints.filterByTag({tags: [tag], property: 'issuers'});
```

Comment:
Are similar updates not required in `20-verify.js` and `30-interop.js` files?

Reply:
Ok so those tests don't use the data-integrity-test-suite assertion so they continue to run as is. I'm not sure if we want to put the effort into updating everything to be uniform in this case as this suite is kind of on the way out.

Reply:
@aljones15 oh okay, I think it's not required and the tests are running fine.